src/app/components/client/ExposureCard.module.scss

```diff
@@ -56,56 +65,92 @@
     a,
     span {
       color: $color-blue-50;
+
+      svg {
+        display: inline; 
+        margin: 0 $spacing-xs;
+      }
     }
 
     &.isOpen {
       border-top: 2px solid rgba($color-purple-70, 0.2);
       overflow: auto;
       height: auto;
+      padding: $spacing-lg;
     }
 
     .exposureListOfExposureTypes {
+      padding-top: $spacing-md;
       display: flex;
       flex-direction: row;
-      gap: $layout-md;
-      align-items: center;
-
-      .fixItBtn {
-        flex-basis: 20%;
+      flex-direction: column;
+      gap: $layout-xs;
 
-        & > * {
-          width: 100%;
-        }
+      @media screen and (min-width: $screen-md) {
+        flex-direction: row;
+        justify-content: space-between;
+        align-items: center;
+        gap: $layout-md;
       }
 
-      ul {
+      dl {
         font: $text-body-sm;
         justify-content: space-between;
-        list-style: none;
         margin-left: 0;
         padding-left: 0;
-        display: flex;
-        align-items: center;
-        gap: $layout-md;
-
-        .detailsFoundItem {
+        display: grid;
+        grid-template-columns: repeat(2, 1fr);
+        grid-gap: $spacing-sm;
+        
+        @media screen and (min-width: $screen-md) {
           display: flex;
+          flex-direction: row;
+          align-items: center;
+        }
 
-          svg {
-            color: $color-purple-70;
+        div + div {
+         padding-left: 23px; // restore the offset of the exposure icon which is -23px
+             
+          @media screen and (min-width: $screen-md) {
+            padding-left: $layout-md;
           }
+        }
 
+        div:first-child {
+          grid-column: 1 / -1;
+        }
+
+        .detailsFoundItem {
           dt {
             display: flex;
             gap: $spacing-sm;
             align-items: center;
+            position: relative;
+
+            .exposureTypeIcon {
+              color: $color-purple-60;
+              position: absolute;
+              margin-top: 2px;
+              margin-left: -23px; // offset the exposure icon
+            }
           }
 
-          dl {
+          dd {
             font: $text-body-xs;
+            font-weight: 600;
           }
         }
       }
     }
   }
 }
+
+.hideOnMobile {
+  display: none !important;
+  visibility: hidden;
+
+  @media screen and (min-width: $screen-md) {
+    display: block !important;
+    visibility: visible;
+  }
+}
```

Comment:
It's also highly recommended to avoid `!important` - there's no coming back from that, as it can't be overridden by more `!important`s. Instead, since this is only applying to a single type of elements anyway, you could instead add this to the `li` of line 43:

```scss
      @media screen and (max-width: $screen-md) {
        &.hideOnMobile {
          display: none;
        }
      }
```

(`visibility:hidden` is also unnecessary when you already have `display: none`.)

That way, the selector is more specific than the plain `li` that sets the `display: flex`, and thus it can override it - and as an added bonus, future readers can infer that from the context.